rafx-framework/src/resources/resource_arc.rs

```diff
@@ -50,7 +50,9 @@ where
     ResourceT: Clone,
 {
     fn drop(&mut self) {
-        self.drop_tx.send(self.resource.clone()).unwrap();
+        if self.drop_tx.send(self.resource.clone()).is_err() {
+            log::warn!("A resource was dropped after the resource manager was destroyed, it might not clean up properly")
```

Comment:
I wonder if we should have an optional feature for "validation-mode". When validation mode is on, this is an expect / panic. When it's off, it's a warn statement. I imagine there are many other places in the code where we'd like to fail fast and loudly during development but relax it for release.

Reply:
made it an expect